### ISSUE TITLE
Add cmd.exe support

### ISSUE THREAD

**Issue by zippy1981** [eleweek/histsync#42]
I'm not sure if you can edit te history of a cmd.exe session,  but that would be a cool feature.

**Comment by raoulvdberge** [comment]
This wouldn't work as histsync uses bash-preexec, which uses bash.

**Comment by zippy1981** [comment]
So ok a different client must be written. It could still talk to the same server that just stores lines of text.

**Comment by raoulvdberge** [comment]
Yes. I don't know if cmd.exe has any kind of API to make this work though.
